lib/download.ts

```diff
@@ -86,6 +86,9 @@ async function downloadVSCodeArchive(options: DownloadOptions) {
 
 	const download = await request.getStream(url);
 	const totalBytes = Number(download.headers['content-length']);
+	const contentType = download.headers['content-type'];
+	const isZip = contentType ? contentType === 'application/zip' : url.endsWith('.zip');
```

Comment:
```suggestion
	const isZip = download.headers['content-type'] === 'application/zip' ?? url.endsWith('.zip');
```

Reply:
`download.headers['content-type'] === 'application/zip'` is always `true` or `false`, so `url.endsWith('.zip')` will never be reached.